src/recoil/atoms/search.ts

```diff
@@ -0,0 +1,20 @@
+import { atom } from 'recoil';
+
+export type SearchProps = {
+  value: string;
+  options: {
+    label: string;
```

Comment:
```suggestion
    label: '그라운드' | '사용자';
```

value 처럼 타입을 제한해서 지정해줄 수 있을 것 같아요!